src/styles/blocks/card.scss

```
@@ -0,0 +1,96 @@
+.container {
+  padding: 50px 40px;
+  margin-bottom: 22px;
+
+  @include page-grid;
+}
+
+.card {
+  width: 166px;
```

it's always better to use `box-sizing: border-box` (apply in * selector - for all elements)

```suggestion
  width: 200px;
```